modules/ROOT/pages/questions/question-05.adoc

```diff
@@ -0,0 +1,12 @@
+5. In *TinyMCE 5.x*, the `paste_data_images` option was set to `false`
```

Comment:
@lnewson TINY-8310

Reply:
> And, if so, should this change in the `paste_data_images` default
   be included in the *Migration Guide*?

Yes, this will need mentioning in the migration guide. 

> When upgrading to 6.0, `paste` goes from being a plugin to being
   Core functionality. Does this, as a consequence, cause the default
   to be changed?

This is not related to it moving to core. This was a separate improvement since it's often enabled anyways. The other reason if I recall correctly for this change was to bring it into line with PowerPaste which does allow local images to be pasted by default. @metricjs might be able to remember more there.

Reply:
Bringing it into line with PowerPaste, and true seeming to be more useful, were the reasons I remember, yes.